product/index.md

```diff
@@ -2,6 +2,17 @@
 
 This covers how we run our business and deliver service to our customers.
 
+## Product Vision
+Our target customers are users familiar with Node-RED looking to run multiple instances or share access across multiple users.
+
+We make it easy to create instances of Node-RED and to share configuration and work between them. We provide collaboration functionality allowing multiple people in a team to build and maintain a flow. 
+We give user a management platform that delivers the features around running node-red such that it requires minimal effort on the part of the customer to configure, update, and maintain their environments.
+We enable customers to run the flows wherever they require based on what systems and devices they are connecting to.
+
+FlowForge is best placed deliver this because our team has extensive experience with the core of Node-RED.
+We differentiate from the competition by offering a range of deployment options for both public shared SaaS and dedicated instances either on-prem or in the customers cloud. Keeping our Node-RED experience consistent with the core Node-RED experience, where there are improvements we feed those back into the project rather than creating custom forks.
```

Comment:
```suggestion
We differentiate from the competition by offering a range of deployment options for both public shared SaaS and dedicated instances either on-prem or in the customers cloud. Keeping our Node-RED experience consistent with the core Node-RED experience. Improvements are fed back into the Node-RED project, rather than creating a fork of the Node-RED project.
```